runtime/compiler/optimizer/J9ValuePropagation.cpp

```diff
@@ -2539,6 +2544,10 @@ J9::ValuePropagation::transformUnflattenedArrayElementLoadStoreUseTypeHint(TR_Op
    // "elseTree" is the fast path that uses regular aaload and aastore to access array elements
    TR::TreeTop *elseTree = TR::TreeTop::create(comp(), callTree->getNode()->duplicateTree());
 
+   if (trace())
+      traceMsg(comp(), "%s: ifTree n%dn (%p). slow path helper call treetop n%dn (%p). fast path treetop n%dn (%p)\n", __FUNCTION__,
+         ifTree->getNode()->getGlobalIndex(), ifTree, thenTree->getNode()->getGlobalIndex(), thenTree, elseTree->getNode()->getGlobalIndex(), elseTree);
```

Comment:
As above, did you really want to print the pointers to the `TR::TreeTop`s here, or did you want to print the pointers to the nodes they contain?  That is, `{if|then|else}Tree` versus `{if|then|else}Tree->getNode()`

Reply:
Fixed it in ef1b11e